### PR TITLE
Use FQCN for SetRegistry/SetCollector

### PR DESCRIPTION
**Pull request by mbrodala**

### CODE REVIEW

File: Documentation/ApiOverview/SiteHandling/SiteSets.rst [hunk 1]
```diff
@@ -392,12 +392,12 @@ already been validated, converted and, if not set, the default value is used.
 SetRegistry
 -----------
 
-The `SetRegistry` retrieves the site sets found in an ordered sequence, as
+The :php:`TYPO3\CMS\Core\Site\Set\SetRegistry` retrieves the site sets found in an ordered sequence, as
 defined by `dependencies` in `config.yaml`. Please preferably use the site
 object to access the required data. However, if you need to query one or more
-site set definitions in order as defined by dependencies, then `SetRegistry`
+site set definitions in order as defined by dependencies, then :short:`SetRegistry`
```
Comment (linawolf) [inline Documentation/ApiOverview/SiteHandling/SiteSets.rst:398]:
```suggestion
site set definitions in order as defined by dependencies, then 
:php-short:`\TYPO3\CMS\Core\Site\Set\SetRegistry`
```

File: Documentation/ApiOverview/SiteHandling/SiteSets.rst [hunk 2]
```diff
@@ -438,7 +438,7 @@ SetCollector
 ~~~~~~~~~~~~
 
 TYPO3 comes with a new `ServiceProvider`, which goes through all extensions
-with the first instantiation of the `SetCollector` and reads all site set
+with the first instantiation of the :short:`SetCollector` and reads all site set
```
Comment (linawolf) [inline Documentation/ApiOverview/SiteHandling/SiteSets.rst:441]:
```suggestion
with the first instantiation of the :php-short:`TYPO3\CMS\Core\Site\Set\SetCollector` and reads all site set
```

File: Documentation/ApiOverview/SiteHandling/SiteSets.rst [hunk 3]
```diff
@@ -450,8 +450,8 @@ definitions found.
 
 However, this is not the official way to access the site set definitions and
 their dependencies. Please access the configuration via the site object.
-Alternatively, you can also use the `SetRegistry`, as only this manages the
+Alternatively, you can also use the :short:`SetRegistry`, as only this manages the
 site sets in the order declared by the dependency specification.
 
-Only use the `SetCollector` if you need to read all site set definitions.
+Only use the :short:`SetCollector` if you need to read all site set definitions.
```
Comment (linawolf) [inline Documentation/ApiOverview/SiteHandling/SiteSets.rst:456]:
```suggestion
Only use the :php-short:`TYPO3\CMS\Core\Site\Set\SetCollector` if you need to read all site set definitions.
```

File: Documentation/ApiOverview/SiteHandling/SiteSets.rst [hunk 4]
```diff
@@ -450,8 +450,8 @@ definitions found.
 
 However, this is not the official way to access the site set definitions and
 their dependencies. Please access the configuration via the site object.
-Alternatively, you can also use the `SetRegistry`, as only this manages the
+Alternatively, you can also use the :short:`SetRegistry`, as only this manages the
```
Comment (linawolf) [inline Documentation/ApiOverview/SiteHandling/SiteSets.rst:453]:
```suggestion
Alternatively, you can also use the :php-short:`\TYPO3\CMS\Core\Site\Set\SetRegistry`, as only this manages the
```

File: Documentation/ApiOverview/SiteHandling/SiteSets.rst [hunk 5]
```diff
@@ -392,12 +392,12 @@ already been validated, converted and, if not set, the default value is used.
 SetRegistry
 -----------
 
-The `SetRegistry` retrieves the site sets found in an ordered sequence, as
+The :php:`TYPO3\CMS\Core\Site\Set\SetRegistry` retrieves the site sets found in an ordered sequence, as
```
Comment (linawolf) [inline Documentation/ApiOverview/SiteHandling/SiteSets.rst:395]:
```suggestion
The :php:`\TYPO3\CMS\Core\Site\Set\SetRegistry` retrieves the site sets found in an ordered sequence, as
```

**Comment by mbrodala** [comment]
Seems like I remembered the `:php-short:` role wrongly. :upside_down_face:

**Comment by linawolf** [comment]
We have docs for that: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Links/Api.html#links-to-the-api-with-the-short-class-name

**Review by linawolf — APPROVED** [review]

**Pull request merged**